index.html

```diff
@@ -44,5 +45,147 @@ <h2>LET’S CONNECT</h2>
+          <span class="tag">css</span>
+          <span class="tag">javaScript</span>
+        </div>
+        <button type="button" class="work-card-seeProject" disabled>See Project</button>
+      </article>
+
+      <article class="work-card">
+        <img src="/media/snapshoot-portfolio-2.png" alt="Work preview for project Tonic" class="work-card-img">
+        <h1 class="work-card-tittle">Tonic</h1>
+        <div class="work-card-info">
+          <h2>Canopy</h2>
+          <span class="circle"></span>
+          <h3>Back End Dev</h3>
+          <span class="circle"></span>
+          <h3>2015</h3>
+        </div>
+        <p class="work-card-description">
+          A daily selection of privately 
+          personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="work-card-tags">
+          <span class="tag">html</span>
+          <span class="tag">css</span>
+          <span class="tag">javaScript</span>
+        </div>
+        <button type="button" class="work-card-seeProject" disabled>See Project</button>
+      </article>
+
+      <article class="work-card">
+        <img src="/media/snapshoot-portfolio-3.png" alt="Work preview for project Multi-Post Stories" class="work-card-img">
+        <h1 class="work-card-tittle">Multi-Post Stories</h1>
+        <div class="work-card-info">
+          <h2>Canopy</h2>
+          <span class="circle"></span>
+          <h3>Back End Dev</h3>
+          <span class="circle"></span>
+          <h3>2015</h3>
+        </div>
+        <p class="work-card-description">
+          A daily selection of privately 
+          personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="work-card-tags">
+          <span class="tag">html</span>
+          <span class="tag">css</span>
+          <span class="tag">javaScript</span>
+        </div>
+        <button type="button" class="work-card-seeProject" disabled>See Project</button>
+      </article>
+    </section>
+
+    <!--about myself section-->
+    <section id="about-myself">
+      <div class="about-myself-top">
+        <h1>About Myself</h1>
+        <p>Hello I’m a software developer! 
+          I can help you build a product , 
+          feature or website Look through some of 
+          my work and experience! If you like 
+          what you see and have a project 
+          you need coded, don’t hestiate to 
+          contact me.</p>
+        <div id="about-myself-social-media">
+          <h2>LET’S CONNECT</h2>
+          <ul class="social-media-container">
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fab fa-github"></i></a></li>
+            <li><a href="#"><i class="fab fa-instagram"></i></a></li>
+          </ul>
+        </div>
+        <button type="button" class="about-myself-resume" disabled>Get my resume</button>
+      </div>
+      <div class="about-myself-skill-list">
+        <nav class="skill-list-menu">
```

Comment:
- [ ] The skill-list-menu, as the name implies should conveniently be a li- tag. Please make the same change for the other skill-list-menu class elements too.

Reply:
:heavy_check_mark: Fixed